### DISCUSSION TITLE
澳洲醫師用3D列印打造「安樂艙」、AI輔助決定生死！背後有哪些爭議？

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#304] [科技新知]
雖然有些人認為這是在設計時尚美化自殺，甚至引發自殺潮，但我很認同文中尼克奇的論點
「它（石棺）真正的意義是要讓死亡的過程更加『民主化』（democratising），」
「我們相信所有理性的成年人都應享有決定要在何時結束生命的權利，這不該是某種握在其他人手上、只有病患才能享有的特權。」
每個人都有自己人生的課題，要怎樣過完自己的一生也只有自己可以決定
當然啦這種事是不能在衝動下做決定的
[https://www.bnext.com.tw/article/66762/sarco-assisted-suicides-device](url)

**Comment by Jie2222** [comment]
文中說到甚至希望能發展成人民可以自行生產、使用，我覺得這樣真的會產生自殺潮，需要有更縝密的管理和評估

**Comment by EricAgumonJou** [comment]
我蠻訝異3D列印搭配AI輔助能做出這麼恐怖的東西（我還是覺得蠻危險可怕的）。
目前看來此款安樂倉在瑞士似乎是合法的。

**Comment by EricAgumonJou** [comment]
前幾年紐西蘭的「自主安樂死」公投案有通過, 
或許未來紐西蘭或其他可以合法安樂死的國家有機會購買該安樂艙。
不過爭議應該還是會不斷延燒下去。

**Reply by EricAgumonJou** [reply]
對於安樂死的議題，我聽我紐西蘭的朋友們說：
安樂死並不是不人道，這是病人的最後選擇。因為安樂死的話，對於患上絕症不能醫的病人來說，能夠用無痛苦的解決生命。
相比起有絕症又不能醫睡在病床慢慢痛苦死去這就是不人道。

**Comment by enoch-lightning** [comment]
我覺得這種東西真的是糟糕透了，發明這種東西的人尤其糟糕。

**Reply by EricAgumonJou** [reply]
如果是重病且無藥可醫的病人，若許對他們來說是一種解脫。
不過還是希望大家好好珍惜自己的性命😅

**Comment by 107602505** [comment]
安樂死一直都是會引起社會大眾關注的議題，我認為目前AI還是著重在科技方面就好，如果病人真的想解脫我想也可以透過別種方法不用花錢買這種商品，實在不太懂這個東西發明的意義為何。

**Comment by cody910120** [comment]
衝動是魔鬼阿你各位千萬別衝動

**Comment by luo910227** [comment]
雖然人有自由意志，但我認為對於全體人類的福祉來說，人們隨意可死亡並不是件好事，所以就算這是何等偉大的結合AI的科技，也不適宜這樣應用。

**Comment by 109802503gloria** [comment]
同意同學的觀點，安樂死有它存在的價值，但普及成人人隨手可得真的不行
